packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -92,6 +94,11 @@ export interface DropdownProps<ItemType>
    */
   ariaLabel?: string;
 
+  /**
+   * Will auto-align the dropdown on first render if it is not visible. This prop is currently experimental and is subject to future changes.
```

Comment:
This comment is now outdated (no longer limited to the first render), here's my take at a better one we could use for every component with an `autoAlign` prop:

```suggestion
   * **Experimental**: Will attempt to automatically align the floating
   * element to avoid collisions with the viewport and being clipped by
   * ancestor elements.